code/modules/research/techweb/all_nodes.dm

```diff
@@ -227,33 +227,42 @@
 	display_name = "Basic Bluespace Theory"
 	description = "Basic studies into the mysterious alternate dimension known as bluespace."
 	prereq_ids = list("base", "datatheory")
-	design_ids = list("beacon", "xenobioconsole", "telesci_gps")
+	design_ids = list("beacon", "xenobioconsole", "telesci_gps", "bluespace_crystal", "xenobio_monkeys")
```

Comment:
PR https://github.com/Citadel-Station-13/Citadel-Station-13/pull/8660
#8660 handles the disks

Reply:
It'll still conflict anyway.

Reply:
or not.